provider/marathon_test.go

```diff
@@ -551,7 +551,7 @@ func TestMarathonTaskFilter(t *testing.T) {
 					},
 				},
 			},
-			expected:         false,
+			expected:         true,
```

Comment:
I think the `specify-port-number` test case (lines 520 ff.) will cover the updated functionality. Could you please update it in the following regards:

1. Change the label port from 80 to 8080 (line 530).
1. Remove the unneeded application ports (complete line 522).